src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -67,9 +69,11 @@ public LinuxAppServiceEventGenerator(LinuxAppServiceFileLoggerFactory loggerFact
         public override void LogFunctionExecutionEvent(string executionId, string siteName, int concurrency, string functionName,
             string invocationId, string executionStage, long executionTimeSpan, bool success)
         {
-            var logger = _loggerFactory.GetOrCreate(FunctionsExecutionEventsCategory);
```

Comment:
I want to point out that LogFunctionExecutionEvent is called by MetricsEventManager at 2 times. Immediately when function executions complete, and every 5 seconds when longer running activities are flushed. So for long running functions, there'll be a 5s delay from MetricsEventManager before it's even flushed to your backoff code.

Reply:
Thank you for bringing this up.  Just re-iterating to make sure that I understand you
When this PR is merged and deployed, the time to first execution event log will be Min(ExecutionTime, 5s).  This could delay scale out slightly for functions with longer execution times.

Reply:
Yes